integration-tests/src/tests/standard_cases/mod.rs

```diff
@@ -1170,7 +1166,7 @@ pub fn test_unstake_while_not_staked(node: impl Node) {
             TESTING_INIT_BALANCE / 2,
         )
         .unwrap();
-    assert_eq!(transaction_result.status, FinalExecutionStatus::SuccessValue(String::new()));
+    assert_eq!(transaction_result.status, FinalExecutionStatus::SuccessValue(Vec::new()));
```

Comment:
This seems like a pretty frequent pattern. I wonder if it would make sense to add an inherent method constructor for the specific case where a success value with an empty vector is being constructed?

Reply:
It only happens in tests though.  Adding test-only function isn’t as appealing to me.